mkdocs_requirements.txt

```diff
@@ -0,0 +1 @@
+pymdown-extensions
```

Comment:
Can you elaborate on mkdocs_requirements.txt? Who is the user of this file? The RTD system when rendering documentation based in mkdocs.yml?

Reply:
Ok. Clear. NTC